src/smart-components/role/role.js

```diff
@@ -22,6 +22,7 @@ import './role.scss';
 const Role = ({ onDelete }) => {
   const history = useHistory();
   const [isDropdownOpen, setDropdownOpen] = useState(false);
+  const [isOCMSystem, setIsOCMSystem] = useState(false);
```

Comment:
I would probably change the name of this variable. Technically the same can apply for other roles than just OCM.

Reply:
hmmm,  nonPermissionAddingRole?